client/package-lock.json

```diff
@@ -2666,98 +2666,6 @@
         "tslint": "^5.20.1",
         "webpack": "^4.0.0",
         "yorkie": "^2.0.0"
-      },
-      "dependencies": {
```

Comment:
This is result of a new `npm i`. Looks like npm just cleaned things up since the last time.